include/envoy/secret/secret_manager.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/secret/secret.h"
+
+namespace Envoy {
+namespace Secret {
+
+/**
+ * A manager for all static secrets.
```

Comment:
Aren't we eventually going to use this for dynamic secrets also? What is the long term plan for this code?

Reply:
We are going to support dynamic secrets. SecretManager will also manage SdsApi clients.
Added TODO comment